utils/src/main/kotlin/Utils.kt

```diff
@@ -198,15 +198,12 @@ fun getPathFromEnvironment(executable: String): File? {
  */
 fun fixupUserHomeProperty() {
     val userHome = System.getProperty("user.home")
-    val checkedUserHome = sequenceOf(
-        userHome,
-        Os.env["HOME"],
-        Os.env["USERPROFILE"]
-    ).find {
-        !it.isNullOrBlank() && it != "?"
+    if (userHome == null || userHome == "?") {
```

Comment:
Why is the '?'-check not needed for the environment variables?

Reply:
Because the JDK bug mentioned in the docs only affects "user.home".